boards/lilygo/ttgo_tbeam/doc/index.rst

```diff
@@ -20,161 +20,44 @@ Some of the ESP32 I/O pins are accessible on the board's pin headers.
-
-To build the sample application using sysbuild use the command:
-
-.. zephyr-app-commands::
-   :tool: west
-   :zephyr-app: samples/hello_world
-   :board: ttgo_tbeam/esp32/procpu
-   :goals: build
-   :west-args: --sysbuild
-   :compact:
-
-By default, the ESP32-PICO-D4 sysbuild creates bootloader (MCUboot) and application
-images. But it can be configured to create other kind of images.
+.. include:: ../../../espressif/common/building-flashing.rst
+   :start-after: espressif-building-flashing
 
-Build directory structure created by sysbuild is different from traditional
-Zephyr build. Output is structured by the domain subdirectories:
+.. include:: ../../../espressif/common/board-variants.rst
+   :start-after: espressif-board-variants
 
-.. code-block::
-
-  build/
-  ├── hello_world
-  │   └── zephyr
-  │       ├── zephyr.elf
-  │       └── zephyr.bin
-  ├── mcuboot
-  │    └── zephyr
-  │       ├── zephyr.elf
-  │       └── zephyr.bin
-  └── domains.yaml
-
-.. note::
-
-   With ``--sysbuild`` option the bootloader will be re-build and re-flash
-   every time the pristine build is used.
-
-For more information about the system build please read the :ref:`sysbuild` documentation.
-
-Manual build
-============
-
-During the development cycle, it is intended to build & flash as quickly possible.
-For that reason, images can be built one at a time using traditional build.
-
-The instructions following are relevant for both manual build and sysbuild.
-The only difference is the structure of the build directory.
-
-.. note::
-
-   Remember that bootloader (MCUboot) needs to be flash at least once.
-
-Build and flash applications as usual (see :ref:`build_an_application` and
-:ref:`application_run` for more details).
-
-.. zephyr-app-commands::
-   :zephyr-app: samples/hello_world
-   :board: ttgo_tbeam/esp32/procpu
-   :goals: build
-
-The usual ``flash`` target will work with the ``ttgo_tbeam`` board target.
-Here is an example for the :zephyr:code-sample:`hello_world`
-application.
-
-.. zephyr-app-commands::
-   :zephyr-app: samples/hello_world
-   :board: ttgo_tbeam/esp32/procpu
-   :goals: flash
-
-The default baud rate for the Lilygo TTGO TBeam is set to 1500000bps. If experiencing issues when flashing,
-try using different values by using ``--esp-baud-rate <BAUD>`` option during
-``west flash`` (e.g. ``west flash --esp-baud-rate 115200``).
-
-You can also open the serial monitor using the following command:
-
-.. code-block:: shell
-
-   west espressif monitor
+Debugging
+=========
 
-After the board has automatically reset and booted, you should see the following
-message in the monitor:
+Lilygo TTGO TBeam debugging is not supported due to pinout limitations.
 
-.. code-block:: console
+Start Application Development
+*****************************
 
-   ***** Booting Zephyr OS vx.x.x-xxx-gxxxxxxxxxxxx *****
-   Hello World! ttgo_tbeam/esp32/procpu
+Before powering up your Lilygo TTGO TBeam, please make sure that the board is in good
```

Comment:
I would just get rid of this.

Reply:
I also thought about it, but since those are the board vendor lines, I chose to let them be as is.